### ISSUE TITLE
2 Credits for newbies: Create a good PPT taking screenshots from main site, referring what tasks to be done, which tasks to be done serially.

### ISSUE THREAD

**Issue by ArchismanKarmakar** [ArchismanKarmakar/business-website-codepeak2021#18]

**Comment by sp-b18ak** [comment]
sir, i would like to resolve this issue

**Comment by sp-b18ak** [comment]
Please assign me and guide regarding this .

Thank you

**Comment by ShivaRapolu01** [comment]
Hello , can I work on this issue?

**Comment by ArchismanKarmakar** [comment]
> Hello , can I work on this issue?

yes go on, remember it's a project so put each and every details with which you need to work on, order which fix after which...

**Comment by re4lvanshsingh** [comment]
could i work on this issue?

**Comment by ArchismanKarmakar** [comment]
> could i work on this issue? @ArchismanKarmakar

go ahead, check out: ![map](https://user-images.githubusercontent.com/92569441/145590394-73a5533a-b4ff-4d18-9097-92d347030174.png)
![ppt](https://github.com/ArchismanKarmakar/business-website-codepeak2021/blob/main/project-ppt.pptx)

**Comment by re4lvanshsingh** [comment]
@ArchismanKarmakar shall i create the issue list for current issues only? or if u have new issues to be added in future then i could add them as well.

**Comment by ArchismanKarmakar** [comment]
> @ArchismanKarmakar shall i create the issue list for current issues only? or if u have new issues to be added in future then i could add them as well.

Yes this project has future issues and you can guess what those are, ig. So you can just give a rough idea of those.
